web/src/stream_muting.js

```diff
@@ -23,6 +24,7 @@ export function update_is_muted(sub, value) {
         } else if (
             narrow_state.is_message_feed_visible() &&
             message_lists.current === message_lists.home &&
+            message_lists.current.selected_row().offset() !== undefined &&
```

Comment:
I included this line of code because the console was throwing a "Cannot read properties of undefined (reading 'top')" error when attempting to unmute an empty stream on the "All Messages" screen.